cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json

```diff
@@ -456,24 +456,9 @@
         "Sub Feature": "trigger csrs",
         "Feature Description": "Some fields in the trigger csrs are hardwired.",
         "Verification Goal": "Check that the tied fields are tied.",
-        "Pass/Fail Criteria": "Self Checking Test",
-        "Test Type": "Directed Self-Checking",
-        "Coverage Method": "Testcase",
-        "Link to Coverage": "TODO csr access test? ",
-        "Review (Marton)": "",
-        "Review (Robin)": "",
-        "Review (Henrik)": ""
-    },
```

Comment:
These red lines must be a case of _"Removed lines in vplan are points where we consider 1 type of coverage to be enough"_ I presume.